HGCalAnalysis/plugins/HGCalAnalysis.cc

```diff
@@ -836,7 +921,75 @@ HGCalAnalysis::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
 		multiclus_slopeY.push_back(multiClusters[i].y());
 		multiclus_cluster2d.push_back(cl2dIndices);
 		multiclus_cl2dSeed.push_back(cl2dSeed);
+		multiclus_firstLay.push_back(*layers.begin());
+		multiclus_lastLay.push_back(*layers.rbegin());
+		multiclus_NLay.push_back(layers.size());
+		if(multiClusters[i].size()<3)
+		  	{
+			multiclus_pcaAxisX.push_back(-2.);
```

Comment:
is -2 a good default value, i.e. will this never happen in reality for the new variables? Maybe use -999?